src/PCN.py

```diff
@@ -0,0 +1,138 @@
+'''
+Base class for PCNs. 
+
+'''
+import networkx as nx
+import MDAnalysis as mda
+import numpy as np
+import random
+
+class PCN():
+	'''
```

Comment:
Do you use tabs? Or 8 spaces? typically in Python we want to use 4 single white spaces

Reply:
I'm using sublime text for python so the spacing is automatic with tabs which is why GitHub reads this as 8 spaces. I've fixed the settings now.